cmd/notation/cert_gen.go

```diff
@@ -96,7 +98,7 @@ func generateTestCert(ctx *cli.Context) error {
 	return nil
 }
 
-func generateTestKey(bits int) (crypto.Signer, []byte, error) {
```

Comment:
We should reuse the code which exist for Unit tests to generate test certs:
https://github.com/notaryproject/notation-core-go/blob/main/testhelper/certificatetest.go.
It might need some refactoring.

Reply:
> We should reuse the code which exist for Unit tests to generate test certs: https://github.com/notaryproject/notation-core-go/blob/main/testhelper/certificatetest.go. It might need some refactoring.

Code updated.